consensus/src/dag/dag_handler.rs

```diff
@@ -67,10 +66,39 @@ impl NetworkHandler {
         _buffer: Vec<DAGMessage>,
     ) -> SyncOutcome {
         // TODO: process buffer
+
+        // TODO: Pass bounded executor from EpochManager.
+        let executor = BoundedExecutor::new(2, Handle::current());
```

Comment:
2 is probably too small, I typically use 8

Reply:
In the next PR, I will pass the executor from epoch manager.